app/assets/javascripts/host_edit.js

```diff
@@ -2,6 +2,47 @@ $(document).on('ContentLoad', function(){onHostEditLoad()});
 $(document).on('AddedClass', function(event, link){load_puppet_class_parameters(link)});
 $(document).on('click', '#params-tab', function() { resizeTextareas($('#params')); });
 
+LookupValueController = function(item) {
```

Comment:
You sure about that?  In production mode it looks like there is a single script and style GET request as expected.